ksqldb-functional-tests/src/test/resources/query-validation-tests/suppress.json

```diff
@@ -3,7 +3,8 @@
     {
       "name": "should emit final result immediately at window end if grace is specified as zero",
       "properties": {
-        "ksql.suppress.enabled": true
+        "ksql.suppress.enabled": true,
```

Comment:
We need new historic plans for these existing QTTs with this change, right? The query plans should've changed to no longer have the suppress.

Reply:
Correct. I did not yet create historical plan on purpose, as it "pollutes" the PR (and I need to wait for the session window KS PR anyway)